Xamarin.Android.sln

```diff
@@ -129,6 +129,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mono.Debugging.Soft", "exte
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "jnienv-gen", "external\Java.Interop\build-tools\jnienv-gen\jnienv-gen.csproj", "{6410DA0F-5E14-4FC0-9AEE-F4C542C96C7A}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "emulator", "emulator", "{977EDCF4-688E-4E49-B4CE-36FE01205FEB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "checkboottimes", "tests\emulator\checkboottimes\checkboottimes.csproj", "{917C2A10-B226-41AB-B775-E1A9E0F29515}"
```

Comment:
Does `checkboottimes` need to be added to *both* `Xamarin.Android.sln` *and* `Xamarin.Android-Tests.sln`?

Reply:
It does not, adding it only on the test.